<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultimediaX</class>
 <widget class="QWidget" name="MultimediaX">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MultimediaX</string>
  </property>
  <widget class="QTextBrowser" name="songInformation">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="openImages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open pics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </widget>
  <widget class="QLabel" name="songName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>321</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>Open file</string>
      </property>
      <property name="iconSize">
       <size>
        <width>34</width>
        <height>16</height>
       </size>
      </property>
      <property name="shortcut">
       <string>Ctrl+O</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playButton">
      <property name="text">
       <string>Play</string>
      </property>
      <property name="iconSize">
       <size>
        <width>34</width>
        <height>14</height>
       </size>
      </property>
      <property name="shortcut">
       <string>P</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="shortcut">
       <string>S</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>openButton</tabstop>
  <tabstop>stopButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>openImages</sender>
   <signal>clicked()</signal>
   <receiver>MultimediaX</receiver>
   <slot>openImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>progressBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>progressBar</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>MultimediaX</receiver>
   <slot>playFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MultimediaX</receiver>
   <slot>stopFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>progressBar</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>test</sender>
   <signal>clicked()</signal>
   <receiver>MultimediaX</receiver>
   <slot>openTestWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>MultimediaX</receiver>
   <slot>openMusic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>testiSignl(int)</signal>
  <slot>openFile()</slot>
  <slot>playFile()</slot>
  <slot>stopFile()</slot>
  <slot>openImages()</slot>
  <slot>openTestWindow()</slot>
  <slot>openMusic()</slot>
 </slots>
</ui>
